roles/api.py

```diff
@@ -0,0 +1,23 @@
+"""
+API for roles
+"""
+from rolepermissions.verifications import has_object_permission
+
+from roles.models import Role
+
+
+def get_advance_searchable_programs(user):
```

Comment:
Meaning, this might be solved also by not exposing the dashboard for not live programs, but I do not know if is is useful to return these users.

Actually we might even solve this by filtering the users at index time...